pypeit/spectrographs/keck_deimos.py

```diff
@@ -394,7 +397,86 @@ def idname(self, ftype):
                  'standard': None,
                  'trace': 'IntFlat' }
         return name[ftype]
-  
+
```

Comment:
Thanks for restructuring this. This is what I thin we want, a set of instrument specific image processing routines that overload generic routines in the generic spectrograph.py file with the same name.